ra/ra.go

```diff
@@ -30,17 +30,6 @@ import (
 	vaPB "github.com/letsencrypt/boulder/va/proto"
 )
 
-// DefaultAuthorizationLifetime is the 10 month default authorization lifetime.
-// When used with a 90-day cert lifetime, this allows creation of certs that will
-// cover a whole year, plus a grace period of a month.
-// TODO(jsha): Read from a config file.
-const DefaultAuthorizationLifetime = 300 * 24 * time.Hour
```

Comment:
It would be nice if we could preserve the explanation of what the authorization lifetime and the pending authorization lifetime are. Maybe this comment & the one for `DefaultPendingAuthorizationLifetime` could be moved to accompany the new fields of the `RA` config struct in `cmd/boulder-ra/main.go`?